pages/new-collective-page.js

```diff
@@ -166,9 +177,21 @@ class NewCollectivePage extends React.Component {
                   isAdmin={Boolean(LoggedInUser && LoggedInUser.canEditCollective(collective))}
                   isRoot={Boolean(LoggedInUser && LoggedInUser.isRoot())}
                   onPrimaryColorChange={onPrimaryColorChange}
+                  step={step}
+                  mode={mode}
                 />
               )}
             </CollectiveThemeProvider>
+            {LoggedInUser && mode === 'onboarding' && parseToBoolean(process.env.ONBOARDING_MODAL) && (
```

Comment:
I don't think we need to check the flag here. If we're there it's because the flag was properly processed elsewhere.

Reply:
Ok, in that case I think I'm misunderstanding how the feature flags work. Where else will we check it?